client/settings/settings-manager/index.js

```diff
@@ -27,6 +31,28 @@ const TABS_CONTENT = [
 ];
 
 const SettingsManager = () => {
+	const { settings, isLoading } = useSettings();
+	const initialSettings = useRef();
```

Comment:
Yeah, but normal usage (in this repo at least) is to import the method directly.

Also, I believe is better to import explicitly only what's needed. By doing so helps with [tree shaking](https://developers.google.com/web/fundamentals/performance/optimizing-javascript/tree-shaking#what_is_tree_shaking).

Reply:
> Also, I believe is better to import explicitly only what's needed. By doing so helps with tree shaking.

That's a good point and I'm glad we're considering tree shaking here 🙂 
But in this case, since we are already importing `React` and `React` contains `useRef` (e.g. `React.useRef()`), I don't think we benefit from tree shaking by importing `useRef` explicitly. At least I don't think we do, I could be wrong.

With that said, this is a non-blocker so no changes are necessary.